src/Action/Generate.hs

```diff
@@ -138,14 +138,21 @@ readHaskellDirs timing settings dirs = do
             yield (name, url, lbstrFromChunks [src])
     return (Map.union
                 (Map.fromList cabals)
-                (Map.fromList $ map ((,mempty{packageTags=[(strPack "set",strPack "all")]}) . fst) packages)
+                (Map.fromList $ map generateBarePackage packages)
            ,Set.fromList $ map fst packages, source)
   where
     parseCabal fp = do
         src <- readFileUTF8' fp
         let pkg = readCabal settings src
         return (strPack $ takeBaseName fp, pkg)
 
+    generateBarePackage (name, dir) =
```

Comment:
In this case `dir` is actually a filename, so could we use `file` here rather than `dir`.

Reply:
👍